example/newfilewithappauth/main.go

```diff
@@ -0,0 +1,107 @@
+/*
+Copyright 2018 The go-github AUTHORS. All rights reserved.
+
+Use of this source code is governed by a BSD-style
+license that can be found in the LICENSE file.
+
```

Comment:
This should be a completely blank line so that the auto-generated godocs correctly add the top-level comment to this example file.

Reply:
Not sure which line you're referring to, we can look after I push changes.